src/engine/ruleset/decoders/sonicwall/firewall-generated.yml

```diff
@@ -0,0 +1,166 @@
+---
+name: decoder/sonicwall-firewall-generated/0
+
+metadata:
+  module: firewall
+  title: Firewall logs decoder
+  description: Decoder for Linux firewall logs
```

Comment:
What is the difference between `generated` and `general`? Maybe having a little brief regarding it would be nice. Also please add the word `Sonicwall` somewhere on the title and description.